apex-app-archetype/src/main/resources/archetype-resources/src/test/resources/log4j.properties

```diff
@@ -1,4 +1,4 @@
-log4j.rootLogger=DEBUG,CONSOLE
+log4j.rootLogger=INFO,CONSOLE
```

Comment:
I don't think we should make this change, especially not in the archetype, where users expect to see their own logging. It is sufficient when you limit the com.datatorrent output below.